hack/jenkins/job-configs/global.yaml

```diff
@@ -13,6 +13,17 @@
     publishers:
         - postbuildscript:
             builders:
+                # Dump any master/node logs that are still around at this point
```

Comment:
Yes. It correctly handles the case where the cluster is torn down and prints something like "Master not detected. Is the cluster up?" and then "Nodes not detected. Is the cluster up?" Then exits without error.